test/builders/transactions.go

```diff
@@ -152,6 +153,18 @@ func TransactionInputArgumentsParse(t *protocol.Transaction) *protocol.ArgumentA
 	return argsArray.ArgumentsIterator()
 }
 
+func TriggerTransaction(pv primitives.ProtocolVersion, vid primitives.VirtualChainId) *protocol.SignedTransaction {
```

Comment:
maybe this shouldn't be strictly a test builder? do you build the actual trigger tx differently?

Reply:
code changed and it is now regular txbuilder with extra.